app/views/hyrax/collections/_search_form.html.erb

```diff
@@ -0,0 +1,12 @@
+  <%= form_for presenter, url: url, method: :get, class: "well form-search" do |f| %>
```

Comment:
This looks like an override to a Hyrax partial.  Could you add the following above the first line:


```ruby
<%# OVERRIDE Hyrax 2.9.6 INSERT THE REASON FOR THE OVERRIDE %>
```